src/uu/touch/src/touch.rs

```diff
@@ -302,3 +310,74 @@ fn parse_timestamp(s: &str) -> UResult<FileTime> {
 
     Ok(ft)
 }
+
+// TODO: this may be a good candidate to put in fsext.rs
+/// Returns a PathBuf to stdout.
+///
+/// On Windows, uses GetFinalPathNameByHandleW to attempt to get the path
+/// from the stdout handle.
+fn pathbuf_from_stdout() -> UResult<PathBuf> {
+    #[cfg(unix)]
+    {
+        Ok(PathBuf::from("/dev/stdout"))
```

Comment:
yeah, the GNU implementation is simpler because they use file descriptors (stdout == 1 on unix and windows) with https://github.com/wertarbyte/coreutils/blob/master/lib/fd-reopen.c but with paths I have no choice but to yank out the path from the stdout file handle object